src/features/multivariate_layer/components/MultivariateAnalysisForm/MultivariateAnalysisForm.tsx

```
@@ -0,0 +1,286 @@
+      className={s.modalDialog}
+      title={i18n.t('multivariate.multivariate_analysis')}
+      onClose={cancelAction}
+      footer={
+        <div className={s.buttonsRow}>
+          {showKeepColorsCheckbox && (
+            <Checkbox
+              id="keepColorsCheckbox"
+              checked={isKeepColorsChecked}
+              onChange={(checked) => setKeepColorsChecked(checked)}
+              label="Keep colors"
+            ></Checkbox>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)